sql/2024/sustainability/cdn_adoption.sql

```diff
@@ -0,0 +1,30 @@
+#standardSQL
+# The distribution of CDN adoption on websites by client.
+
+SELECT
+  client,
+  IF(cdn = '', 'No CDN', cdn) AS cdn,
+  COUNT(0) AS freq,
+  total,
+  COUNT(0) / total AS pct
+FROM (
+  SELECT
+    client,
+    COUNT(0) AS total,
+    ARRAY_CONCAT_AGG(SPLIT(JSON_EXTRACT_SCALAR(summary, '$.cdn'), ', ')) AS cdn_list
+  FROM
+    `httparchive.all.pages`
+  WHERE
+    date = '2024-06-01'
```

Comment:
Is it worth adding ` is_root_page` here? This is especially true since the CDN is likely applied to all pages, so no need to look at the primary/secondary pages differently.